decksite/static/css/pd.css

```diff
@@ -859,9 +863,8 @@ p.bugged {
 }
 
 
-.bugged::after {
-    content: "🐞 ";
-    margin-left: 0.2em;
+.bugicon {
+    float:left;
```

Comment:
If we don't clear anywhere does this cause cards with shorter names to bunch up more-than-one on a line?

Do we still need the margin-left to stop the bug icon touching the associated card? We want a small gap there.

Reply:
There was already a clear:both in the .decklist that I thought would be enough, I'm not sure. I'll add the clear:both to the li too, just in case.

Reply:
And you were right too with the margin-left :)